packages/tables/resources/lang/es/table.php

```diff
@@ -35,7 +35,7 @@
 
         'label' => 'Navegación de paginación',
 
-        'overview' => 'Mostrando :first a :last de :total resultados',
+        'overview' => 'Se muestran de :first a :last de :total resultados',
```

Comment:
We should keep the different translations for 1 or many, since it will read really weird otherwise. I made it less verbose now. Could you do a final check if this is correct before I merge it, @pathros?

```suggestion
        'overview' => '{1} Se muestra un resultado|[2,*] Se muestran de :first a :last de :total resultados',
        
```